### DISCUSSION TITLE
set openvpn

### DISCUSSION THREAD

**Discussion by moren75** [RaspAP/raspap-webgui#751] [General]
Hi, I don't understand how to attach the file provided by my vpn provider in the .ovpn box?

**Comment by billz** [comment]
1. Browse to your provider's .ovpn file and choose **Save settings**
2. Confirm that the OpenVPN client.conf uploaded successfully
3. Choose **Start OpenVPN**

See the video walkthrough below.

https://user-images.githubusercontent.com/229399/104086753-d6403b00-525a-11eb-9734-ad7e3ddb22bb.mov

**Comment by moren75** [comment]
Many thanks for video


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Da: Bill Zimmerman <notifications@github.com>
Inviato: Saturday, January 9, 2021 9:19:36 AM
A: billz/raspap-webgui <raspap-webgui@noreply.github.com>
Cc: moren75 <moreno.nardi75@gmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [billz/raspap-webgui] set openvpn (#751)


  1.  Browse to your provider's .ovpn file and choose Save settings
  2.  Confirm that the OpenVPN client.conf uploaded successfully
  3.  Choose Start OpenVPN

See the video walkthrough below.

https://user-images.githubusercontent.com/229399/104086753-d6403b00-525a-11eb-9734-ad7e3ddb22bb.mov

**Comment by billz** [comment]
There is now a page dedicated to OpenVPN in the docs https://docs.raspap.com/openvpn/

This includes the above video walkthrough, plus some additional notes about mitigating DNS leaks.

**Comment by moren75** [comment]
Thanks for update

Moreno Nardi
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Da: Bill Zimmerman <notifications@github.com>
Inviato: Wednesday, January 27, 2021 9:58:33 AM
A: billz/raspap-webgui <raspap-webgui@noreply.github.com>
Cc: moren75 <moreno.nardi75@gmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [billz/raspap-webgui] set openvpn (#751)


There is now a page dedicated to OpenVPN in the docs https://docs.raspap.com/openvpn/

This includes the above video walkthrough, plus some additional notes about mitigating DNS leaks.
